Tools/LyTestTools/ly_test_tools/o3de/settings.py

```diff
@@ -31,15 +31,15 @@ def get_temp_path(self):
         return self._temp_path
 
     def modify_asset_processor_setting(self, setting, value):
-        logger.info(f'Updating setting {setting} to {value}')
```

Comment:
These destructive actions should remain logged at info or higher, however I think the majority of this file is dead code and should be deprecated.